src/memote/suite/tests/test_consistency.py

```diff
@@ -51,32 +51,82 @@ def test_stoichiometric_consistency(model):
     publication. In addition, the list of min unconservable sets is computed
     using the algorithm described in section 3.3.
 
+    """
+    ann = test_stoichiometric_consistency.annotation
+    is_consistent = consistency.check_stoichiometric_consistency(model)
+    ann["data"] = is_consistent
+    ann["metric"] = 1.0 - float(is_consistent)
+    ann["message"] = wrapper.fill(
+        """This model is stoichiomerically {}""".format(
+            "consistent" if is_consistent else "inconsistent"
+        )
+    )
+    assert is_consistent, ann["message"]
+
+
+@annotate(title="Uncoserved Metabolites", format_type="percent")
+def test_unconserved_metabolites(model):
+    """
+    Report number of unconserved metabolites (if model incosistent).
+
+    List of unconserved metabolites is computed using the algorithm described
+    in section 3.2 of
+    "Detection of Stoichiometric Inconsistencies in Biomolecular Models."
+    Bioinformatics 24, no. 19 (2008): 2245.
+    doi: 10.1093/bioinformatics/btn425.
     """
     ann = test_stoichiometric_consistency.annotation
     is_consistent = consistency.check_stoichiometric_consistency(model)
     ann["data"] = {
         "unconserved_metabolites": []
         if is_consistent
         else get_ids(consistency.find_unconserved_metabolites(model)),
+    }
```

Comment:
Now that we have separate tests, this can go directly into data, I think.

```suggestion
    if is_consistent:
        ann["data"] = []
    else:
        ann["data"] = get_ids(consistency.find_unconserved_metabolites(model))
```

Reply:
Please adjust the rest of the test accordingly.